### PR TITLE
Solve Received compressed data elegantly

### PR DESCRIPTION
**Pull request by rendonghai**
#105

### CODE REVIEW

File: befh/exchange.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 from befh.market_data import L2Depth, Trade, Snapshot
 from datetime import datetime
 from threading import Lock
-
+from befh.util import Logger
```
Comment (gavincyi) [inline befh/exchange.py]: Unused import.

File: befh/sqlite_client.py [hunk 2]
```diff
@@ -32,6 +34,7 @@ def execute(self, sql):
         Execute the sql command
         :param sql: SQL command
         """
+        Logger.info(self.__class__.__name__, 'Command: {}'.format(sql))
```
Comment (gavincyi) [inline befh/sqlite_client.py]: Unnecessary logging.

File: befh/util.py [hunk 3]
```diff
@@ -11,8 +11,9 @@ def init_log(output=None):
         Initialise the logger
         """
         Logger.logger = logging.getLogger('BitcoinExchangeFH')
-        Logger.logger.setLevel(logging.ERROR)
-        Logger.logger.setLevel(logging.INFO)
+        Logger.logger.setLevel(logging.DEBUG)
```
Comment (gavincyi) [inline befh/util.py]: Why need to set a DEBUG level?

Comment (rendonghai) [reply befh/util.py]: I saw you just set the log level for 2 times. I just remove one and set the debug as default level

File: befh/sqlite_client.py (hunk 2 above)
Comment (rendonghai) [reply befh/sqlite_client.py]: It could be removed.

File: befh/exchange.py (hunk 1 above)
Comment (rendonghai) [reply befh/exchange.py]: Just for debugging this part.

**Comment by rendonghai** [comment]
I have reverted unnecessary logging steps and add many instruments is this commit.
Please review and merge this pull request.

And reminder that #107  issue should be fixed as soon as possible. It will affect all the Restful instrument with processes for fetching l2 depth and trades data.

**Comment by gavincyi** [comment]
All looks good to me in general but I don't have any time to review in details recently. So I merge it first and will review it later. Thanks a lot for your great contribution. It's wonderful.

**Pull request merged**